components/core/src/service.rs

```diff
@@ -386,6 +386,10 @@ impl Into<u64> for HealthCheckInterval {
     fn into(self) -> u64 { self.0.as_secs() }
 }
 
+impl Into<Option<Duration>> for HealthCheckInterval {
```

Comment:
I think this can be removed in favor of the existing conversion [here](https://github.com/habitat-sh/habitat/blob/c1e104258c73ddbace9c567deda347f46583dfac/components/core/src/service.rs#L405).